package/yast2-packager.spec

```diff
@@ -62,6 +65,9 @@ Requires:       /usr/bin/md5sum
 # .process agent
 Requires:       yast2-core >= 2.16.35
 
+# Storage (methods to check the devicegraph)
+Requires:       libstorage-ng-ruby
```

Comment:
well, I am not sure if it is good idea to allow other modules to use directly libstorage. In past we also do not do it to use underlaying stuff. I hope yast2-storage provide all methods needed.